packages/app/src/components/Visualization/Visualization.js

```diff
@@ -9,11 +9,11 @@ import VisualizationPlugin from '@dhis2/data-visualizer-plugin'
 import { sGetVisualization } from '../../reducers/visualization'
 import { sGetCurrent } from '../../reducers/current'
 import { sGetUiRightSidebarOpen, sGetUiInterpretation } from '../../reducers/ui'
-import { sGetLoadError } from '../../reducers/loader'
+import { sGetLoadError, sGetIsPluginLoading } from '../../reducers/loader'
```

Comment:
I noticed that I'm mixing `IsPluginLoading` and `PluginLoading`, should be consistent, but which is preferred?